Cargo.toml

```diff
@@ -99,7 +99,7 @@ thiserror = "1"
 tokio = { version = "1.39", default-features = false }
 tokio-test = "0.4"
 # Fix the version of uuid to prevent build breaking
-uuid = "=1.12.1"
+uuid = {version = "1.15.1", features = ["js"]}
```

Comment:
The features should be added at the crate level and only for appropriate targets.